brainreg_segment/plugins.py

```diff
@@ -1,7 +1,20 @@
 from napari_plugin_engine import napari_hook_implementation
 from brainreg_segment.segment import SegmentationWidget
-
+from brainreg_segment.measurement import measurement
+from brainreg_segment.measurement.layers import (generate_random_points_2d_layer,
```

Comment:
Should this be importing from `random_layers` rather than `layers`?

Reply:
Yes you are right, fixed it